src/rgw/CMakeLists.txt

```diff
@@ -260,4 +265,9 @@ if(WITH_EMBEDDED)
   if(WITH_RADOSGW_FCGI_FRONTEND) 
     target_link_libraries(cephd_rgw ${FCGI_LIBRARY})
   endif()
+  if(WITH_RADOSGW_BEAST_FRONTEND)
+    target_link_libraries(cephd_rgw LINK_PRIVATE
+      ${Boost_COROUTINE_LIBRARY}
+      ${Boost_CONTEXT_LIBRARY})
+  endif()
```

Comment:
Yup. I built with `./do_cmake.sh -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib64 -DCMAKE_INSTALL_LIBEXECDIR=/usr/lib -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_MANDIR=/usr/share/man -DCMAKE_INSTALL_DOCDIR=/usr/share/doc/packages/ceph -DWITH_EMBEDDED=ON -DWITH_MANPAGE=ON -DWITH_PYTHON3=ON -DWITH_SYSTEMD=ON -DWITH_TESTS=OFF -DWITH_LTTNG=ON -DHAVE_BABELTRACE=ON -DWITH_OCF=ON -DWITH_RADOSGW_BEAST_FRONTEND=ON -DBOOST_J=8` and it succeeds without this block.